docs/lsp-clients.json

```diff
@@ -425,23 +425,31 @@
   },
   {
     "name": "jedi",
-    "full-name": "Python(Jedi-language-server)",
+    "full-name": "Python (Jedi-language-server)",
     "server-name": "jedi",
     "server-url": "https://github.com/pappasam/jedi-language-server",
     "installation-url": "https://github.com/fredcamps/lsp-jedi",
     "debugger": "Not available"
   },
   {
-    "name": "pyls",
+    "name": "pylsp",
     "full-name": "Python",
+    "server-name": "pylsp",
+    "server-url": "https://github.com/python-lsp/python-lsp-server",
+    "installation": "pip install 'python-lsp-server[all]'",
+    "debugger": "Yes"
+  },
+  {
+    "name": "pyls",
+    "full-name": "Python (Palantir)",
```

Comment:
can you put deprecated in the name or in the full-name. You may add entry in changelog.org.

Reply:
Done, also added to the changelog